WordPress/WordPressTest/Extensions/XCTestCase+Wait.swift

```diff
@@ -7,6 +7,6 @@ extension XCTestCase {
             waitExpectation.fulfill()
         }
 
-        waitForExpectations(timeout: duration)
+        waitForExpectations(timeout: duration + 1)
```

Comment:
Thank you! 🙌 
Wdyt about leaving a clarifying comment?

Reply:
Done in ca9bfbbe17ac91bfacbe34ff089d5d4438187fb6 👍